src/Fixer/Basic/BracesFixer.php

```diff
@@ -30,17 +30,26 @@
  */
 final class BracesFixer extends AbstractFixer implements ConfigurableFixerInterface, WhitespacesAwareFixerInterface
 {
+    const NEXT_LINE = 'next';
+
+    const SAME_LINE = 'same';
+
+    private $supportedBracePositions = array(self::NEXT_LINE, self::SAME_LINE);
+
     /**
      * @var array
      */
     private $configuration;
 
     private static $defaultConfiguration = array(
         'allow_single_line_closure' => false,
+        'position_after_functions_and_oop_constructs' => self::NEXT_LINE,
     );
 
     /**
-     * @param array<string, bool>|null $configuration
+     * @param array|null $configuration
```

Comment:
because of the the new config option it changed to
```php
/** @param array<string, string|bool> $configuration */
```

so it needs to be updated to something ;)